rules/proto_rules.build_defs

```diff
@@ -483,7 +491,6 @@ def protoc_binary(name, url:str|dict = '', version:str = '', hashes=None, deps=N
         cmd = '$TOOL x $SRCS --strip_prefix include',
         labels = [
             'protoc:-I"$TMP_DIR"/' + package_name(),
-            'protoc:-I"$TMP_DIR"',
```

Comment:
We add `-I.` to all protoc invocations so this isn't needed and was actually causing issues.